presidential-elections/utils/model.py

```diff
@@ -39,45 +39,50 @@ class PresidentialElectionsModel:
 
     """
 
+    political_families = [
+        "farleft",
+        "left",
+        "green",
+        "center",
+        "right",
+        "farright",
+        "souverainistes",
```

Comment:
Are you down to rename "souverainistes" to "identitaires" in this PR?

Reply:
It looks like `souverainistes` is used in the file with the election results as well. Should be open a separate issue?

Reply:
Yes please